Runtime/Scripts/PersistentConnection.cs

```diff
@@ -442,24 +542,23 @@ async void PropagateSuccess(FindCloudletEventTrigger trigger, FindCloudletReply
       }
       if (config.autoMigration)
       {
-        await StopEdgeEvents();
+        StopEdgeEvents();
         integration.persistentConnection.startStreamingEvents(integration);
       }
       else
       {
         //Migration occurs and autoMigration is set to false
-        await StopEdgeEvents();
+        StopEdgeEvents();
```

Comment:
This should be handled by the app, since it need to transfer state under its control.

Reply:
Now logging that Migration occurs and asking developer to StopEdgeEvents on their own.